src/v2/Components/NavBar/NavBar.tsx

```diff
@@ -143,172 +173,155 @@ export const NavBar: React.FC = track(
+                      {...anchorProps}
+                    >
+                      <NavBarItemUnfocusableAnchor href="/collect" />
+                      Artworks
+                    </NavBarItemButton>
+                  )}
+                </Dropdown>
+
+                <NavBarItemLink href="/auctions">Auctions</NavBarItemLink>
+
+                <NavBarItemLink href="/viewing-rooms">
+                  Viewing&nbsp;Rooms
+                </NavBarItemLink>
 
-                <NavItem href="/auctions">Auctions</NavItem>
-                <NavItem href="/viewing-rooms">Viewing&nbsp;Rooms</NavItem>
-                <NavItem href="/galleries">Galleries</NavItem>
-                <NavItem href="/fairs">Fairs</NavItem>
-                <NavItem href="/Shows">Shows</NavItem>
-                <NavItem
+                <NavBarItemLink href="/galleries">Galleries</NavBarItemLink>
+
+                <NavBarItemLink href="/fairs">Fairs</NavBarItemLink>
+
+                <NavBarItemLink href="/Shows">Shows</NavBarItemLink>
+
+                <NavBarItemLink
                   // Hide link at smaller viewports — corresponding display inside of `MoreNavMenu`
                   // If we need to do this again, consider a more abstract solution
                   display={["none", "none", "flex", "flex"]}
                   href="/institutions"
                 >
                   Museums
-                </NavItem>
-              </NavSection>
-              <NavSection alignItems="right" mr={2}>
-                <ScrollIntoView
-                  selector="#download-app-banner"
-                  behavior="smooth"
+                </NavBarItemLink>
+              </Flex>
+
+              <Flex alignItems="stretch" display={["none", "none", "flex"]}>
+                <NavBarItemButton
+                  px={0}
+                  pl={1}
+                  onClick={() => {
+                    scrollIntoView({
+                      selector: "#download-app-banner",
+                      behavior: "smooth",
+                    })
+                  }}
                 >
-                  <NavItem
-                    href="#download-app-banner"
-                    onClick={event => event.preventDefault()}
-                  >
-                    Download App
-                  </NavItem>
-                </ScrollIntoView>
-              </NavSection>
-            </NavSection>
-          </NavBarTier>
+                  Download App
+                </NavBarItemButton>
+              </Flex>
+            </Text>
+          </HorizontalPadding>
+        </AppContainer>
+      </Box>
 
-          {showMobileMenu && (
-            <>
-              <MobileNavMenu
-                onClose={() => toggleMobileNav(false)}
-                isOpen={showMobileMenu}
-                onNavButtonClick={handleMobileNavClick}
-              />
-            </>
-          )}
-        </NavBarContainer>
-      </header>
-    </>
+      {/* TODO: V3 */}
```

Comment:
This is actually fine/looks identical since v2 text variants still work in v3. But there is a bunch of updates that I'll make here in a separate PR.